path/src/update.ps1

```
@@ -110,15 +119,43 @@ function script:fount_update_to_ref($Target) {
 	deno_upgrade
 }
 
-# After the first successful deno upgrade, routine starts refresh in the background.
+# `fount update <remote-url>` —— 把 origin 指到该远程并切到其默认分支（随后普通更新跟随它）。
+function script:fount_update_to_url($Url) {
+	if (invoke_repo_git remote 2>$null -contains 'origin') {
```

PSScriptAnalyzer (1.25.0)

[warning] Did you mean to use the redirection operator '>'? The comparison operators in PowerShell are '-gt' (greater than) or '-ge' (greater or equal).

(PSPossibleIncorrectUsageOfRedirectionOperator)